checks/evaluation/contributors_test.go

```diff
@@ -20,134 +20,75 @@ import (
+						Probe:   "contributorsFromOrgOrCompany",
+						Outcome: finding.OutcomePositive,
 					},
 					{
-						NumContributions: 10,
-						Organizations: []clients.User{
-							{
-								Login: "org2",
-							},
-						},
-					},
-					{
-						NumContributions: 10,
-						Organizations: []clients.User{
-							{
-								Login: "org3",
-							},
-						},
-					},
-					{
-						NumContributions: 1,
-						Organizations: []clients.User{
-							{
-								Login: "org2",
-							},
-						},
+						Probe:   "contributorsFromOrgOrCompany",
+						Outcome: finding.OutcomePositive,
 					},
 				},
 			},
-			expected: checker.CheckResult{
+			want: checker.CheckResult{
+				Score:   0,
 				Version: 2,
-				Score:   10,
-				Reason:  "4 different organizations found -- score normalized to 10",
+				Name:    "contributorsFromOrgOrCompany",
+				Reason:  "project has 2 contributing companies or organizations",
 			},
-		},
-		{
-			name: "Contributors with orgs and number of contributions is greater than 5 without companies",
-			raw: &checker.ContributorsData{
-				Users: []clients.User{
-					{
-						NumContributions: 10,
-						Organizations: []clients.User{
-							{
-								Login: "org1",
-							},
-						},
-					},
+		}, {
+			name: "Has two positive outcomes",
+			args: args{
+				name: "Contributors",
+				findings: []finding.Finding{
 					{
-						NumContributions: 10,
-						Organizations: []clients.User{
-							{
-								Login: "org2",
-							},
-						},
+						Probe:   "contributorsFromOrgOrCompany",
+						Outcome: finding.OutcomePositive,
 					},
 					{
-						NumContributions: 10,
-						Organizations: []clients.User{
-							{
-								Login: "org3",
-							},
-						},
+						Probe:   "contributorsFromOrgOrCompany",
+						Outcome: finding.OutcomePositive,
 					},
 					{
-						NumContributions: 1,
-						Organizations: []clients.User{
-							{
-								Login: "org10",
-							},
-						},
+						Probe:   "contributorsFromOrgOrCompany",
+						Outcome: finding.OutcomePositive,
 					},
 				},
 			},
-			expected: checker.CheckResult{
-				Version: 2,
+			want: checker.CheckResult{
 				Score:   10,
-				Reason:  "3 different organizations found -- score normalized to 10",
+				Version: 2,
+				Name:    "contributorsFromOrgOrCompany",
+				Reason:  "project has 3 contributing companies or organizations",
 			},
 		},
 	}
-	for _, tc := range testCases {
-		tc := tc
-		t.Run(tc.name, func(t *testing.T) {
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
-			result := Contributors("", &utests.TestDetailLogger{}, tc.raw)
-			if !cmp.Equal(result, tc.expected, cmpopts.IgnoreFields(checker.CheckResult{}, "Error")) { //nolint:govet
-				t.Errorf("expected %v, got %v", tc.expected, cmp.Diff(tc.expected, result, cmpopts.IgnoreFields(checker.CheckResult{}, "Error"))) //nolint:lll
+			result := Contributors("contributorsFromOrgOrCompany", tt.args.findings)
```

Comment:
let's just re-use t.name as first param of Contributor, instead of hardcoding a probe name.